src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <!--Logo-->
+      <a href="#home">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+      <!--Main Nav-->
+      <nav class="navigation">
+        <ul class="main-nav">
+          <li>
+            <a class="nav-link is-active" 
+              href="#apple">APPLE</a>
+          </li>
```

Comment:
Need to fix it everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">APPLE</a>
          </li>
          
```